lib/rules/no-debugging-utils.ts

```diff
@@ -13,14 +13,22 @@ import {
 } from '../node-utils';
 import { DEBUG_UTILS } from '../utils';
 
-type DebugUtilsToCheckFor = Partial<
-	Record<typeof DEBUG_UTILS[number], boolean>
->;
+type DebugUtilsToCheckForConfig = Record<typeof DEBUG_UTILS[number], boolean>;
+type DebugUtilsToCheckFor = Partial<DebugUtilsToCheckForConfig>;
 
 export const RULE_NAME = 'no-debugging-utils';
 export type MessageIds = 'noDebug';
 type Options = [{ utilsToCheckFor?: DebugUtilsToCheckFor }];
 
+const defaultUtilsToCheckFor: DebugUtilsToCheckForConfig = {
```

Comment:
Intentionally defined a variable with this type so if a debug method is removed/added a type error will be thrown for this default config so it remains in sync

Reply:
Good idea!